src/aiida/brokers/rabbitmq/broker.py

```
@@ -98,15 +98,28 @@ def __str__(self) -> str:
+    def probe_service_status(self) -> BrokerServiceStatus:
         """Return status information reported by the RabbitMQ server."""
-        properties = self.get_communicator().server_properties
-        return {
-            key: t.cast(JsonValue, value.decode('utf-8') if isinstance(value, bytes) else value)
-            for key, value in properties.items()
-        }
+        had_communicator = self._communicator is not None
+
+        try:
+            properties = self.get_communicator().server_properties
+        except Exception as exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)